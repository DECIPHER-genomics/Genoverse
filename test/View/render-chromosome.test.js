const { afterTest, testTrackRenderStatic } = require('../utils');

describe('Correctly render a chromosome:', () => {
  afterEach(afterTest);

  const genome = {
    '1': {
      size  : 1000,
      bands : [
        { id: 'p13',    start: 1,    end: 100,  type: 'gpos'    },
        { id: 'p12',    start: 101,  end: 250,  type: 'stalk'   },
        { id: 'p11.2',  start: 251,  end: 300,  type: 'gneg'    },
        { id: 'p11.1',  start: 301,  end: 350,  type: 'acen'    },
        { id: 'q11.1',  start: 351,  end: 450,  type: 'acen'    },
        { id: 'q11.21', start: 451,  end: 500,  type: 'gpos100' },
        { id: 'q11.22', start: 501,  end: 600,  type: 'gpos25'  },
        { id: 'q11.23', start: 601,  end: 700,  type: 'gpos33'  },
        { id: 'q12.1',  start: 701,  end: 800,  type: 'gpos50'  },
        { id: 'q12.2',  start: 801,  end: 900,  type: 'gpos66'  },
        { id: 'q12.3',  start: 901,  end: 1000, type: 'gvar'    },
      ],
    },
    '2': {
      size  : 1000,
      bands : [{ id: 'p13', start: 1, end: 1000, type: 'gpos' }],
    },
  };

  it('with all band types', () => testTrackRenderStatic(
    undefined,
    { _testClass: Genoverse.Track.Chromosome, featureHeight: 20 },
    { end: genome[1].size, genome: genome, chromosomeSize: genome[1].size, width: genome[1].size }
  ));

  it('with only one band', () => testTrackRenderStatic(
    undefined,
    { _testClass: Genoverse.Track.Chromosome, featureHeight: 20 },
    { chr: 2, end: genome[2].size, genome: genome, chromosomeSize: genome[2].size, width: genome[2].size }
  ));
});
