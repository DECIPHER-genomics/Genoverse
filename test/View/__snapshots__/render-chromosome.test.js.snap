// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly render a chromosome: with all band types 1`] = `"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA+gAAAAVCAYAAADVR8S+AAAABmJLR0QA/wD/AP+gvaeTAAAZnElEQVR4nO3de3AT170H8K+kXb1lSbZlyy+MjXmZQBMCDqQBuwVSGBrACbTkNuZZbgoJLRDftkwJBJLemfT2Fjo3w0zTBNKQJuUxaZpAeYSmPExxSHBCCcYQgx+yrYclS7JkS6t93T+EjI1tbBxjyfb5zGiYOTq7Pout1f72d85vJbhNBeC3AP4DgAFEVEhlMoaSUW5IJVU8y17lef46gJu3XuUAAtEdIRGjaAAZALIBZFMUNYaSKx6AKI7iONbMsaw2yuPrFxRF+6Qy6ibPhUp4nr8A4CKAqwCEKA+NiE0qACkAUiP/0grFSJqms0VRkiYKnJljOQPHsZroDpMgCIIgiOFIrVZ7pFLpu36/vxi34jzJrfekAE4CmIbwBQ0RJWt/8d/wul1tL7erkWly2hlvk0sWaPWppFJZkKLpekD8mgkGvwJwA7cD+JvRHT1xnxlxKwAHkE3L5eNlMmo8L/CZXChkklE0q9MbGENCkiwhMVmtj0+Q6I2J0BsTsGfXjigPvX/8Z/EO2Bvq4Gq0Cfb6Wp+9vgatLX4FrVDc4HnuHMsw5xAO2isA8FEeLnH/GNEu6AaQQisUWTIZNVIUxVSIopFlQyaIokyh1gQ1mjhOp9dLdfp4ZZzeKNfo4qDR6aHRxkGpUuGP/7stukfTT65duxbtIRDtNDQ0RHsIRDvXr1+P9hCIdsjnI7asXLky2kMYtqxWK7Zt2xY4f/58qc/nmw1AoG69twzAVJDgPOoUKjWSVGokpWa0Nd16QeB5+Jo9aq/bNdrrdo72Nrm+53Lagx6Xg/d53UqB56UymnbJpLIalmOv8Cx7DST7Pph0yIJLpdLRtFyeC0hGcyybJoqCXKnSMvr4BD4+MVllTDDRemMC9MYExJvMoGi67W9lqNLo9Mgeq0f22AlSAHoAYIIBuBzWCfYGywRrXXWRra6Ga/E1K2i5vJbnuS/YUCgStH8GgInm+Im7UgBIwO2g24h2gTeAdJ7nk7gQkyCjKFal1jJanV7QxhmoOEOCWhsXJ1Wq1G2Bt05vhFyhAIAhMXuEIAiCIIihJyUlBbt371YtWLBg6rVr15bxPP9WJEB/GeQiJuZJZTJEAjJgDADIALRNzWQCrfC6Xcm3Xnkk+x6T+pwFD//eQd1178OQQqlC6ohspI7IxkPIVwFAiAnCaW/IsTdYcqx11d+31dVw/maPiqblVlEUPw2FmBKEg/YyAK1RPYChT4VbwTZuZbspihohpehMmUyWKQpCCsdxJoHnNLRcHlJrdKxWp4dKq5PH6Y1KjU4v0er0iGS9dXojpFLpkL8ZRRAEQRDE8EBRFLZt26ZdvXr1y36//y0KwIMIXzwRgxzJvscEkgWPAXKFslPQzvM8PK7GDIfVkmG1VM9rsFSzbqddQ8vlDqlUVtba0vIJIHyFcNDeFN0jGBQ6re+mKCpDRsuzpRJJhiAIZo5jE0RBoGm5glVrtCGtTg+dIZ6O0xtVmji9RKPVQ6uLg0YXB7U2DhKJhNyAIgiCIAhi2HnkkUegUqkMfr//WxSA2SBZuSGvp+x7sC377gxn3532VrfTEfI0NVHBgF8tiqI0WmMfTCQSiaBUaVsN8fGcMTFJbkxMVof/38NZcKVKDZDPW1TIZDIkJJmRkGTG+G9N1QDhG1duV2Oaw2pJs9bVPG61VDNNjVa1RCr1UxR1KRgInAbwOYBSAI1RPYCB06f13XH6eKXujvXdWl0cFLf/5skSKoIgCIIgiG7MnTuX/vOf/zyHAlCAQZyxe+edd/C9730PNpsNEydOBBC+EC8tLUViYiJkMhkOHjyI4uJiiKIY5dHGLqVKDaVKjeTb2Xf1rRdxb6QY4stFnnnmGRw/fhxmsxmXL1/usT3CYrFg5cqVqKiogEKhwMaNG/H8888P5NA7kXYM2hUAFIIgwO10GBxWS36jzTrd0VDrtzfUqkWIQYqiLgUZ5jQE4XOE17TbonoAvXfn+u5IRXNzb9d3R6aZK5RqxBmMoOVkfXdEV99Dd2uPyMjIwN69ezFu3DgwDIOdO3fitdde69W2g1VxcTFKSkqQmJiIw4cP99geYbVasXnzZty8eRNyuRwrVqzAM88802070Tu//vWv8dlnnyE+Ph579uzpsT3C4XDg1VdfRW1tLWiaxpIlS1BYWAhBELBu3Tp4vV4IgoD8/HysXbsWEomk0z6Iu/vud7+L9PR0tLa24tChQwAArVaL/Px8GAwG8DyPy5cv48qVK73alvhmnnzySeTk5MDv92P37t0AAL1ej4ULFyIxMRE8z+P8+fO4cOFCh+1604e4dxs2bMCZM2dgMplw/PhxAOHvieLiYty4cQNyuRyrV6/G8uXLO2zH8zwKCwvR1NQEnucxf/58/OpXv4qZc9TkyZMVH330UYEEwBUAudEeUF8VFBSgpaUFe/bsabuIkUgkSE5Ohs1mg1KpxCeffIKXX34ZR48ejfJoe7bhpV3RHgIxBO3c9rN+29epU6eg0WiwatWqDoF4d+0RFosFVVVVmDlzJhwOByZPnowTJ04gN7f3p5+N23/fL8fQFy0+L+wNdbA31PJWS1Wrrb6WFgUhRCsUX4WCgfMcx/0b4XXt5QAG6m5g+2nmRvRyfbfOEE/r9EaVUqXuan33AA09dux6acM32r6r76G7tUdkZGQgKysLZ86cQVJSEsrKyvD444+jvLy8x227MhiquF+4cAEqlQqbN2/uEIh31x5htVpRV1eHqVOnwuVyobCwEHv27IFGo+myPScnZyAPq0uDoUr1l19+CaVSid/85jcdAvHu2iMcDgdsNhsmTZoEj8eDNWvW4Le//S1GjBgBt9uN+Ph4hEIhbNq0CUVFRXjkkUcG8rC6NNiquKempoJlWeTn53cI0HU6HaxWK1QqFZ588kn8/e9/h9vt7nHbWDMYPh/tjRw5EizLYuHChR0CdIPBgJqaGmg0Gjz77LPYt28fGhtvT7jrTZ9YMNiquJeWlkKlUuHnP/95hwDdYrEgLy8PLpcL8+fPx759+zB69Oi27URRhNPphMlkAsMwePrpp/HTn/4UBQUF0TqUDi5evIgf//jHVygMgvXnZrMZp0+fxj//+U8kJCRAKpVixYoV8Pl8OHXqVKcvYlEUYbOFE1symQxSqZRkzwniHthsNuTn5+M73/kOXC4XBEHAW2+9BZ1Oh4KCAlRWVnbaprv2iIyMDGRkhGdoJCUlYdy4cWhoaLinAD2a2lWQlwHQAUCLz6u0N9Q9aq+vnWatq2qxN1gojmNB0fJKLsSe5Xn2U/TtWe1kfXcM2rRpE9asWQOLxYLq6mqUl5dj165dXX4PAei2PcJiscBisQAIBzwVFRVITU1FeXl5j9vGur179+LAgQNISUlBWloacnJysHz5cuTl5aGmpqZT/+7aI1JSUpCSkgIASEhIQHZ2NhwOBx599NEu2wfz/939cPDgQRw+fBhJSUkwm83IzMzE4sWL8eCDD6K+vr5T/+7aI5KSkpCUlAQAMBgMGDFiBJxOJzIzMxEfHw8AEAQBgiDETGYqVk2aNAnjxo2D3++H3++H2+3G5cuX0dDQgLi4uA59I30AIBAIwOPxQK1WdwrQu9qW6J3p06fj4YcfhtfrhcfjQWNjI0pLS1FdXd32tx3h9Xrh9XoBAC0tLXA6ndDpdB2C7970Ibr3xhtv4L333kNqairS09MxevRorFq1CtOmTUN1dXWHvnd+T4waNQp2u71DgC6RSGAymQCEs+mxdo4yGo3gOC6eAjAoPsGjRo3CggULcO3aNfziF7/Axo0bsWPH3Z/tfP36dWRkZOCdd97BiRMnBmikBDE03LhxAx9++CHGjh2LV199FTt37sTWrVv7Zd/l5eW4efMmpk2b1i/7i5Y7HvsWCdrhsNZNtFvrJlgtVUX2BouUDTFSpVJVybGhswzDlCI8NVwAYKbl8hEymSwLosTM8Vwyz7F6iqYZpUoT0urieK3OQOmM8UpdnEGh0cVBo9VDo9VBG6cHRcsBsr57QIwfPx6rV6/GlClTIIoiLly4gPLy8n7bf25uLrKzs1FaWtpv+4yWGzdu4NChQ3j//fcBAEuWLOnXgLmyshIWiwUPPvhgr9qHu5qaGhw5cgR/+MMfIJFI8JOf/ASZmZn9un+r1drhZmtRUREcDgfmzJmDKVOm9NvPGmoMBgPGjh3b9lkpLCzsFGx3x2g0Ii4uDg6H434OcVgxmUyYPHkyXn/9dYiiiDVr1vQ6kDaZTDAajairq/tGfYjbKisrsX//fnz00UeQSCRYuHBhh2D7br7++mvU1tbioYce6vL9goICWK1WFBYWYsaMGf057G/EaDQiFArpKADKaA+mN6qqqtqm7x05cgQ7d+7scZsxY8bAYDDg/fffR15e3pC48CGIgZKVlYWxY8cCAObPn4+NGzf2y37dbjeefvpp7N69G1rt0FvCrNHpkaXTI2vMBGmz26WzNdSi+no5d+3KFxN4jpsoo6iFoihqJRIJzXO8ig2FJCwApUrDZYzM4ZNSM2R6Y4JardWpdXEGaLRxUKo1Pf5c4v769re/jcOHD6OlpQUA2i6o+4PRaMR7772HdevWtWXHBrOysjIUFBRApQrfN5ozZ06/7bu5uRmbNm3CSy+9BLVa3WM7AXz11VeYPn06lMrw5d7MmTP7bd8+nw87duzAhg0b2n7fALBv3z74/X5s3boVFRUVg2am1EAzm82ora0Fx3EAwte6vaFQKDBr1iyUlJSAZdn7OcRhJSMjA9evX0coFAIAXL16tVfbqVQqLF68GEeOHGnbti99iI4+//xzzJo1q+2cPnfu3F5t5/V6sX79erzyyivQaLq+fjp16hSam5vx7LPP4tKlS90G8gNNp9OBZVkVhfDjs2L+Kpmibs/OpGm611PWPR4Pjh8/jieeeIIE6ARxDyIXDADAsmy/TAEKBAJYtGgRfvazn/X6RDtYtK1Rr68VrHVVLe3XqAdbWz4RBOECgFKe4+68HW8EkBoMtKRUfV2eXXOjIoOSK0b2tIb8zqns+vhEKJQkkX6/3Pmd019T4lQqFT744AP8/ve/x7Fjx/pln9F2v6YLBoNBrFu3DsuXL++Q8eiunQi78/fRX0v+GIbBli1bsHjxYuTl5XV6X6vVYurUqfjXv/5FAvR+RFEUHn/8cVy+fLltiQxxf/TmXEbTNJYuXYrS0tJul/n1pg/RWV/OXcFgEGvWrMGqVauQn59/175xcXGYOXMmTp48GTMBenNzM+RyeYAC4MMgCNBHjhyJxx57DCUlJVi2bBnOnj3bbd/k5GQolUrU1NTAaDRiwYIFbQUdCILonerqapSUlOCxxx7D22+/3ecL3w8++ABz584FRVFYunQp5s6di1WrVvXzaAcWEwzC5WiArcEiNNTc9NXX3JSFQgFZeO05c4bn+dMAzgKwsWyPd8rdt15XgPC6zVAwcGcfBRMMJjDBYIrb1ZiKdsXgpBLJGFEUUyNr0nt69Fm7CuzEPTh37hxeeOEFvPzyyxBFEYWFhXjjjTf6tK9Fixbh2LFj4DgOf/nLX3Ds2LEuC3ENVpMnT8aePXvw3HPPQSKR4OOPP8aSJUv6tK+TJ09ixowZoCgKGzduxIwZM/DUU0+1vc/zfJftxG0PPPAADhw4gGXLlkEikeDs2bOYP39+n/ZVUlKCvLw8yGQy7NixA3l5eZg3b17b+263G6FQCMnJyfD5fDh37hwWLVrUX4cy5EQK7ZWVlUEURWRlZaGioqLb/lKpFLNmzYLFYulUFHLkyJGwWCzgef5+D3vIqq2txfTp03H69GkAwLhx41BWVtZtf6lUisWLF6OyshJffPFFh/fGjRuHyspKCILQbR/i7qZMmYI//vGPWL9+PSQSCY4fP46lS5d225/jOKxfvx4FBQX4wQ9+0OG9EydOID8/Hz6fDwzDIC0tDV6vFydPnkRRUdH9PpRec7vdoGm6WQLgKwAToj2guzGbzfj4449RWlqKvLw8VFVVoaioCD6fD/v378f06dORnJyMuro6vPLKKzh37hwOHToEk8mEYDCId999F1u2bBkUheJIFXfifrjXKu42mw1z5szBtGnTcOHCBWRlZWHfvn3Q6XT44Q9/iPPnz8NutyM9PR1btmzBypUru203Go24evUqysvLMXv2bGRlZbX9nN/97ndYuHBhr8cVjSruIYaB014Pe4MF1rrqgNVSzbb6fXJKLv/6jkJwA1m9vTs9FpeLPMNcqdIyemM829Wj1IZbRffeVnHfuHEj1q5dC4vFgsrKSly9ehW7du3q8nto79693ba73W6MHz8eubm5OHnyZIdprZs2bcLf/va3bre9m1iq4v7WW2/h3XffRUpKCjIzMzFq1CgsX74cGzZswJdffgmn0wmz2Yy1a9fiqaee6rZ96tSpOHr0KCorK7FixQqkp6e3/YzNmzdDo9F02T5r1qxoHHYHsVSl+uDBg/jwww9hMpmQlpbWViRu+/btKC8vR1NTE0wmE4qKijBv3rxu25944gn86U9/Qk1NDV544QWYzea2n/Hcc88hMzMT27Ztg8fjgVwux6xZs7B69eqYKMIUq1XcJ06ciNzcXLS0tLQVJrt8+TJmz56NpKQkqNVq+P1+fPHFF/D5fPj+97+P5ubmtu0jBcxWrFiBAwcOoLW1tcttY+n8AMTW56O96dOnY8qUKWhubkZTU1NbkbglS5YgPT0dWq0Wzc3NOHPmDDweD5YtWwaPx9O2/fHjx1FRUYFf/vKXeO2112AymbrtE0titYr7m2++iX379iE1NRUjR45ETk4OVq1aheeffx4XL16E0+lESkoK1q9fj7S0NPzoRz9qK0gMAC+++CLmzJmDSZMm4R//+Af8fj/Wrl0Ll8sFhUKBhQsXori4OCbOUUB4Wv+aNWu+kgA4DKBvt1IHiNlsxtGjR2Nm+sH9FM0AnQ0xaHI60NRoQ1OjjXdY6/yuRru0xefViKI4PK7UvyGJRCJodPqWBFOykJSSro03mWXxJjPiE5OimrXsS4A+b968mLvbe78DdDYUQqOtri0Yt9XVcC2+ZoVCqbzBcuxZlmHOoW9V2WONEe2CeAAptEKR1dMz0fXGeI0+PrGrCvFRPZhvqi+PWSsuLgbHcdi1K3ZuqsbaBXjEm2++CYqiOj2PdqiL1QBk//79kMlkWLx4cbSHMqBiNUBvb9KkSRBFsctHlQ41sfr5aO/RRx+FIAjDYolsrAbo7b3++uugKGrQz8K8m7/+9a/Yvn37EQrAKQCzAZA5j8MEE2iF1+2Cq9GGRnsD09hgCboaHVQw4FfQcoVNIpVeCQYDlyAIVxCedluOcK0CogeiKNL+Zk+Gv9kzoebGtVy5UvWAVCp9KMQEc6RSGQzxpkByaroiMTlVlZCUAr0xAXpjQrSHPSwJPA+3qxEOqwVWS3VLveUm43Y6dDRF2wRRKGVDoXMASgB82driH2pzBjtMqwcAlmHAgunQiWNZo8/rTvV53ZFsfLqUosff8/p4YwIUKlK4iyAIgiAIojtlZWVMc3PzKQmAbwE4B4CUCY4B/ZlBb/F54Wq0w+t2otHawDTa6gJNTpuc4zgJTcvrReAyEwheAIQbCAfhgz0rGOtSAeQCyKYVismUjHqYZdnRgKjQxRmYxOQ0WXJqukZvTJTEm8yINyX3W2byXjPosaqvGXRBEOB2OsLBeF0NY7VUM02NNiVFUXZRFD8NhZgShDPjnwF3RKlETxQAEtAuGx9ZHy+TyTJFEWlsKJjSm/XxOr0RcsXA3yvuSwY9FsVqBn24GgwZwuFkMGTQhxPy+YgtgyGDPhxMnTrV39jY+G0KwCWEMykkQB+EBJ6Hr9kDl8MWnprutLc6rPUht9OuAcDSCkWVwPFloVDwIsLZspsAbvLtKnQTA6bh1uvObKXR7WrMdrsaJ1TfuPoQRdEPcxw7hmfZBI1O35qYnIIEk1mXkJQiSTCZEW8yg6LpqB3EYNDi86Khtgp11ZWhBkt1sKnRqpZStJum6NLWlpZPAOFzABd5niMzQ745Brf/ti8Ct54A0Pkcowr4fSkBvy/Vaa+/6/p4hVoT1OuNIZ0+Xj7c18cTBEEQBDH0ffrppwgEAh4A/448u+xFAP+HQVDNfbjiOQ6eJieaGm1w2q2Cw14f8DjtXLPHrZJSlFcmlVaxHHuRZ9kvEc6GXwLg41tJID4IuBEObC6yDPM2y7QF7gqf153j87pzayorJtJy+cMSiXRMKMSMUKjUIYMxUUhKSVclJqfQCSYzEpPTopJ9jLbI481s9TWhhpqbfntDrVqEGKQo6lIwEDgN4HMAJTzPu1kmGO3hDmcB3LpBGGngOK7D4/xuiQTy2bb62ntaH6/R6SVanR56Y8KQWR9PEARBEMTQxnEctm/f7g8EAi8CQCRAfxvAMgDTEK4CTERJX9aH8zwXYKM9cOJ+YBD+HV8RBOEgE2wLLumA35cR8PsmWC1VvVrnPlR0ftZ4DS0KYoiSyy8FAq3/hCB8DuA8ACfHkk/FINUpkO/N+ngAKbRcPp6i5dn3sj5eo4sb0IMjCIIgCIKIsFqt2Lp1a8BisXzG8/zbANA+taAC8D8AfgTAEI0BEgBFy5spmq4VBOFSKBgoQzgbXgGgBtF/hBMR2yQAMgGMA5ArV6omS6XSb3EsO4JjQ0MiCqHlyiZKJrvCsMwpgeNKEc6OO6I9LiJmKXF7bXwawuvjM2m5MhsQRwiCkMyxbDzPc8roDpMgCIIgiOFIrVZ7pFLpn/1+/3/hVlHu/weco5LIqNj3zQAAAABJRU5ErkJggg=="`;

exports[`Correctly render a chromosome: with only one band 1`] = `"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA+gAAAAVCAYAAADVR8S+AAAABmJLR0QA/wD/AP+gvaeTAAAC/0lEQVR4nO3dz0ubdxzA8XceE0Y3NudCMR6KhkiunZeATOkGHYhbvLmjP447Cr3uTxC97CbbRQTPTkQPGhFK0SGTiKAgASlTMIxsdpMxSHZQ1xbWKLbpt5D36/Z8nu8D7xzD9/skMZ67A3wPfAN8gCRJkiRJapQKMAc8As4BYpc3IuAxkHthJkmSJEmSGucceAI8BKotl8Mx4FsuvqhLkiRJkqTGSwB3gV+BX652y8tAMliSJEmSJEnN6ylwLwI+BVoDx0iSJEmS1Kw+Bu5HXJx1jweOkSRJkiSpWSWALyPg69AlkiTpdmZnZzk9PaVYLP43a2lpYWtri1KpxNHREZOTk8Ri/gasJEnvsPeAzyMgHbpEkiTdzszMDIODgy/NqtUq+XyedDpNNpult7eXgYGBQIWSJOmGuuLAR6ErJEnSq6VSKdbX11lbWyOZTBJFEWNjY5ydnVEoFOju7n5pfa1W4+TkBLjYTY+iiFqtFiJdkiTd3CcR8H7oCkmSVF8mk2Fqaorh4WE2NzeZmJi49pmDgwPK5TLFYpGVlZW3UClJkl7DhxEXL6NLkqR3WKlUYn9/H4DFxUX6+/uvfSabzdLR0UEmkyGXyzU6UZIkvZ47EfBP6ApJklRfPP78D1cSicSNj6xXKhWWl5fJ5/ONSpMkSW/GeQT8FbpCkiTV19XVRV9fHwAjIyNsbGy8cm17ezudnZ0AtLW1MTQ0xN7e3lvplCRJt/ZHBPweukKSJNW3u7vL6OgoOzs7pNNppqenAZifn2d1dZVsNsvh4SHj4+O0traysLDA8fEx29vbFAoF5ubmAn8CSZJ0jd9iQAF4ELpEkiT9v1QqxdLSEj09PaFTJElS4yxGwE+hKyRJkiRJamJ/A4UYcB/4GYjXXy9JkiRJkhrgGfBZ7PKiDCQDxkiSJEmS1KyeAveiy4tHQDVgjCRJkiRJzegZ8B3A1Q56BDwGci/MJEmSJElS45wDT4CHQPVqB70KfAH8CPwZKEySJEmSpGZRAX4AvuLyRPu/12mio8TqluwAAAAASUVORK5CYII="`;
